packages/master.cfg.py

```diff
@@ -0,0 +1,55 @@
+import multiprocessing
+import json
+from pathlib import Path
+from buildbot_nix import (
+    NixConfigurator,
+    BuildbotNixConfig,
+)
+from buildbot.process.factory import BuildFactory
+
+factory = BuildFactory()
+
+STATE_DIR = Path(".")
+PORT = 8012
+
+buildbot_nix_config = BuildbotNixConfig(
+    db_url="sqlite:///state.sqlite",
+    auth_backend="none",
+    gitea=None,
+    github=None,
+    pull_based=dict(
```

Comment:
There is room for improvements in this file, but I kind of want to have some basic thing working because the current iteration cycles are kind of long.